types/react-native-modern-datepicker/index.d.ts

```diff
@@ -0,0 +1,46 @@
+// Type definitions for react-native-modern-datepicker 1.0
+// Project: https://github.com/HosseinShabani/react-native-modern-datepicker#readme
+// Definitions by: Ankan002 <https://github.com/Ankan002>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { CSSProperties } from 'react';
+
+export interface ModernDatepickerProps {
+    onSelectedChange?: (dateString: string) => void;
+    onMonthYearChange?: (dateString: string) => void;
+    onTimeChange?: (dateString: string) => void;
+    onDateChange?: (dateString: string) => void;
+    minimumDate?: string;
+    maximumDate?: string;
+    selectorStartingYear?: number;
+    selectorEndingYear?: number;
+    disableDateChange?: boolean;
+    isGregorian?: boolean;
+    mode?: 'datepicker' | 'calender' | 'monthYear' | 'time';
+    minuteInterval?: 1 | 2 | 3 | 4 | 5 | 6 | 10 | 12 | 15 | 20 | 30 | 60;
+    style?: CSSProperties;
+    current?: string;
+    selected?: string;
+    options?: {
+        baclgroundColor?: string;
+        textHeaderColor?: string;
+        textDefaultColor?: string;
+        selectedTextColor?: string;
+        mainColor?: string;
+        textSecondaryColor?: string;
+        borderColor?: string;
+        defaultFont?: string;
+        headerFont?: string;
+        textFontSize?: number;
+        textHeaderFontSize?: number;
+        headerAnimationDistance?: number;
+        daysAnimationDistance?: number;
+    };
+}
+
+export default function(props: ModernDatepickerProps): JSX.Element;
+export function getToday(): string;
+export function getMonthName(month: number): string;
+export function getFormatedDate(date?: Date, format?: string): string;
+export function toPersianNumber(value: string): string;
+export function toEnglish(value: string): string;
```

Comment:
```suggestion
export function getToday(): string;
export function getFormatedDate(date?: Date, format?: string): string;
```
there are named exports in the module file, only couple of symbols are being exported IMO:
https://github.com/HosseinShabani/react-native-modern-datepicker/blob/master/src/index.js#L4-L5

Reply:
Yeah that's right my fault will rectify and  rectify this PR. Actually I misundertood the utils file. Sorry for that.

Reply:
Hey @peterblazejewicz I have just now pushed the required changes to fix the issue of extra functions. Please review once you have time.